mandatory/1-fix-functions.js

```diff
@@ -10,18 +10,19 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function getMood(isHappy) {
+   if (isHappy === true) {
```

Comment:
here, you are checking if a boolean === true for the if statement - you could just have if(isHappy) :)